lib/existing-eks-opensource-observability-pattern/index.ts

```diff
@@ -39,12 +40,35 @@ export default class ExistingEksOpenSourceobservabilityPattern {
         });
 
         // All Grafana Dashboard URLs from `cdk.json` if presentgi
-        const clusterDashUrl: string = utils.valueFromContext(scope, "cluster.dashboard.url", undefined);
-        const kubeletDashUrl: string = utils.valueFromContext(scope, "kubelet.dashboard.url", undefined);
-        const namespaceWorkloadsDashUrl: string = utils.valueFromContext(scope, "namespaceworkloads.dashboard.url", undefined);
-        const nodeExporterDashUrl: string = utils.valueFromContext(scope, "nodeexporter.dashboard.url", undefined);
-        const nodesDashUrl: string = utils.valueFromContext(scope, "nodes.dashboard.url", undefined);
-        const workloadsDashUrl: string = utils.valueFromContext(scope, "workloads.dashboard.url", undefined);
+        const fluxRepository: blueprints.FluxGitRepo = utils.valueFromContext(scope, "fluxRepository", undefined);
+        fluxRepository.values!.AMG_AWS_REGION = region;
+        fluxRepository.values!.AMP_ENDPOINT_URL = ampEndpoint;
+        fluxRepository.values!.AMG_ENDPOINT_URL = amgEndpointUrl;
+
+        const ampAddOnProps: blueprints.AmpAddOnProps = {
+            ampPrometheusEndpoint: ampEndpoint,
+            ampRules: {
+                ampWorkspaceArn: ampWorkspaceArn,
+                ruleFilePaths: [
+                    __dirname + '/../common/resources/amp-config/alerting-rules.yml',
+                    __dirname + '/../common/resources/amp-config/recording-rules.yml'
+                ]
+            }
+        };
+
+        if (utils.valueFromContext(scope, "java.pattern.enabled", false)) {
```

Comment:
`(utils.valueFromContext(scope, "java.pattern.enabled", undefined)` situation is not handled. I think people may ignore this in few patterns.

Reply:
false is the default value is not found. I guess no change here right?

Reply:
I think i agree.